custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -1,43 +1,189 @@
+            <div id="infrastructure" ng-show="expandedSectionId==='infrastructure'">
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/clean_water') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/clean_water', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported Clean Drinking Water</a>
+                </div>
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/functional_toilet') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/functional_toilet', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported functional toilet</a>
+                </div>
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/infantometer') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/infantometer', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported Infantometer</a>
+                </div>
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/stadiometer') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/stadiometer', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported Stadiometer</a>
+                </div>
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/medicine_kit') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/medicine_kit', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported medicine kits</a>
+                </div>
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/infants_weight_scale') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/infants_weight_scale', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported Weighing Scale: Infants</a>
+                </div>
+                <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-class="{'active-mobile-link': $location.path().indexOf('/awc_infrastructure/adult_weight_scale') !== -1}">
+                    <a href="{$ goToStep('awc_infrastructure/adult_weight_scale', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
+                      AWCs Reported mother and child weighing scale</a>
```

Comment:
I think some of these titles need to be reconciled with the latest changes on web. See, e.g.
https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/templates/icds_reports/icds_app/navigation.directive.html#L124

Reply:
Sure. Will update it